### ISSUE TITLE
Destroy method making two options :selected

### ISSUE THREAD

**Issue by kaspiCZ** [selectize/selectize.js#827]
https://jsfiddle.net/zp6q0vdg/

The first option of a `<select>` is selected, but when `destroy()` is called, the second option in the set gets `selected: true` in the DOM too.

I have tracked it down to line 1284 in the fiddle's JS where a `$.detach()` is called. The workaround I used is commented out. What happens is that the `$.detach()` modifies the original `$.children()` set. That said, it is probably a jQuery bug and I'll have further look into it to report it.

It might be up for consideration to use `$.clone()` and `$.remove()` as the workaround in the fiddle suggests. It is questionable whether we want to preserve the event bindings as well, because the jQuery documentation suggests `$.detach()` keeps only jQuery data (I'll test later if that is truly the case). But I assume we do, because we want to revert the element to its initial state.

**Comment by risadams** [comment]
closing stale issues older than one year.
If this issue was closed in error please message the maintainers.
All issues must include a proper title, description, and examples.
